bundles/org.openhab.binding.gce/src/main/resources/ESH-INF/thing/ipx800v3.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="gce" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="ipx800v3">
+		<label>IPX800v3</label>
+		<description>The GCE IPX800v3 device</description>
+
+		<channel-groups>
+			<channel-group id="I" typeId="I"/>
```

Comment:
Only comment I have left is that in general type are lower or camel case. These id's here are also very short. Does that make sense to the user? Especially I an O can be difficult to distinguish from 1 and 0.

Reply:
I could rethink this. I've been lazy enough to use the PLC coding as channel groups. I could for sure make it more readable :)